### PR TITLE
Currency: Updated librdkafka scripts with minimal dependencies

### PR DESCRIPTION
**Pull request by Balavva-Mirji**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Have you validated script on UBI 9 container
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: l/librdkafka/librdkafka_ubi_8.10.sh [hunk 1]
```diff
@@ -41,15 +41,22 @@ if ! ./configure --install-deps ; then
...
-    echo "------------------$PACKAGE_NAME:install_fails---------------------"
+    echo "------------------$PACKAGE_NAME: install_fails---------------------"
     echo "$PACKAGE_URL $PACKAGE_NAME"
     echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail |  Install_Fails"
     exit 1
 fi
 
 if ! make -C tests run_local_quick; then
-    echo "------------------$PACKAGE_NAME:install_fails---------------------"
+    echo "------------------$PACKAGE_NAME: install_success---------------------"
```
Comment (prachikurade1) [inline l/librdkafka/librdkafka_ubi_8.10.sh:59]: can be changed to install_success_but_test_fails

File: l/librdkafka/librdkafka_ubi_9.3.sh [hunk 2]
```diff
@@ -41,15 +41,22 @@ if ! ./configure --install-deps ; then
...
-    echo "------------------$PACKAGE_NAME:install_fails---------------------"
+    echo "------------------$PACKAGE_NAME: install_fails---------------------"
     echo "$PACKAGE_URL $PACKAGE_NAME"
     echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail |  Install_Fails"
     exit 1
 fi
 
 if ! make -C tests run_local_quick; then
-    echo "------------------$PACKAGE_NAME:install_fails---------------------"
+    echo "------------------$PACKAGE_NAME: install_success---------------------"
```
Comment (prachikurade1) [inline l/librdkafka/librdkafka_ubi_9.3.sh:59]: can be changed to install_success_but_test_fails

**Review by prachikurade1 — CHANGES_REQUESTED** [review]

File: l/librdkafka/librdkafka_ubi_9.3.sh (hunk 2 above)
Comment (Balavva-Mirji) [reply l/librdkafka/librdkafka_ubi_9.3.sh]: @prachikurade1 done with the required changes.

**Review by prachikurade1 — APPROVED** [review]
Approved

**Pull request merged**